pbm/delete.go

```
@@ -84,9 +127,9 @@ func (p *PBM) DeleteOlderThan(t time.Time) error {
 			return errors.Wrap(err, "decode backup meta")
 		}
 
-		err = probeDelete(m)
+		err = p.probeDelete(m)
 		if err != nil {
-			return err
```

I think it's worth to log than just ignoring error